game-rules.md

```diff
@@ -142,14 +142,17 @@ A utility building that provides **{Energy generated per turn}** extra energy pe
 * Missiles move at least one cell the moment it is created.
 
 ## Score
-Each player will have a score based on the value of the buildings that the player destroyed of his opponent.
-**Note that a building that is not fully built will reward the full amount of points.**
+Each player will have a score based on the damage dealt by the player on both the opponent's health and buildings, as
+well as the players own energy generated and buildings built. 
+**Note that a building that is not fully built will reward the full amount of points to the player destroying it.**
 
-Scores are calculated by adding:
+Scores are calculated by adding the following together:
 * Total damage dealt
 * Fixed score for each building constructed
 * Total energy generated
-* End game HP x 100
+* Damage to the opponents health per hit x 100
+
+Note that each time the opponents health pool is damaged, the player receives 100 points per hit.
```

Comment:
the player receives points equal to damage dealt times 100.

Hit count does not affect score